packages/skeleton-react/src/lib/components/AppBar/AppBar.test.tsx

```diff
@@ -0,0 +1,196 @@
+import { describe, expect, it } from 'vitest';
+import { render } from '@testing-library/react';
+import { AppBar } from '$lib/index.js';
+
+describe('AppBar', () => {
+	describe('<AppBar>', () => {
+		const testId = 'app-bar';
+		it('Renders the component', () => {
+			const component = render(<AppBar></AppBar>);
+			expect(component.getByTestId(testId)).toBeInTheDocument();
+		});
+
+		it('Renders the children', () => {
+			const component = render(
+				<AppBar>
+					<div data-testid="child"></div>
+				</AppBar>
+			);
+			expect(component.getByTestId('child')).toBeInTheDocument();
+		});
+
+		for (const prop of ['base', 'background', 'spaceY', 'border', 'padding', 'shadow', 'classes']) {
+			it(`Correctly applies the \`${prop}\` prop`, () => {
```

Comment:
Are the slash escapes needed if these are template literals? Seems redundant?

Reply:
They are needed to get a nicely formatted test like this:
Progress > <Progress> > Meter > Correctly applies the `meterAnimate` prop based on the `value` prop